src/icons/OpenLinkIcon.tsx

```diff
@@ -4,15 +4,15 @@ import colors from 'src/styles/colors'
 
 interface Props {
   color?: colors
+  size?: number
 }
 
-const OpenLinkIcon = ({ color = colors.white }: Props) => (
-  <Svg width={12} height={12} fill="none">
+const OpenLinkIcon = ({ color = colors.white, size = 12 }: Props) => (
+  <Svg width={size} height={size} fill="none" viewBox="0 0 12 12">
```

Comment:
should the viewBox use `size` instead of 12?

Reply:
no, this won't scale the image correctly. The viewBox has to be fixed at 12 for the scaling to work